Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -344,8 +349,47 @@ public async Task<TeamViewModel> UpdateTeamAsync(int teamID, TeamPatchViewModel
 
             return t;
         }
+
+        private async Task SendInviteEmail(int teamID, string inviteeUsername, string? inviterUsername)
+        {
+            var team = _context.Team.FirstOrDefault(t => t.ID == teamID);
+            var activity = _context.Activity.FirstOrDefault(a => a.ID == team.ActivityID);
+            var invitee = _accountService.GetAccountByUsername(inviteeUsername);
+            var inviter = _accountService.GetAccountByUsername(inviterUsername);
```

Comment:
Is inviterUsername required here? What happens if it isn't passed?

It looks like the email below will contain blank information. Something like "  has invited you to join a team".

Reply:
There isn't a use case where that will happen. 

Currently the only people who get that email are SPECIFICALLY people who get invited (aka their initial assigned role type is "Requested Join" or "Invited" don't remember what it was exactly, but the ID=2) 

Captains/Founders are assigned DIRECTLY to their role ID of 4 or 5, can't remember which

Rec-IM Admins can make them any ID they want, so if they so desire to make them pending, then it would say that the Admin invited them.